src/discordtipbot/crystal/string.cr

```diff
@@ -0,0 +1,6 @@
+class String
+  # Splits given string like required for parsing commands
+  def cmd_split
```

Comment:
A little overkill to reopen.. just using `string.split(' ', remove_empty: true)` everywhere would be a simpler diff.

If you actually want to DRY this, `split` the string early-on and pass the already-split `args` array around.

Reply:
I think I'll eventually handlr this using Middleware und Just merge as is

Reply:
That's a good option too. Go ahead then if you want, just seemed to be a more complicated solution than necessary.